RecoPPS/ProtonReconstruction/src/ProtonReconstructionAlgorithm.cc

```diff
@@ -169,7 +169,7 @@ double ProtonReconstructionAlgorithm::newtonGoalFcn(double xi,
 //----------------------------------------------------------------------------------------------------
 
 reco::ForwardProton ProtonReconstructionAlgorithm::reconstructFromMultiRP(const CTPPSLocalTrackLiteRefVector &tracks,
-                                                                          const LHCInfo &lhcInfo,
+                                                                          const float energy,
```

Comment:
I'd encode the units in the variable name (if that is not against CMSSW coding rules), renaming `energy` to `energy_GeV` or `energy_TeV`

Reply:
energy should be in GeV in CMSSW, if it's not then indeed better to include it in the name

Reply:
in PPS reconstruction code we use GeV, like in most places of CMSSW. I've made that suggestion to be more explicit, but we can stick to the GeV-convention as well.